build-scripts/run/start.bat

```diff
@@ -11,10 +11,11 @@ call start.frontend.bat nopause
 
 pushd ..\..
 docker-compose --env-file %ENVFILE% -p dq-d7 -f docker/cli.yml run --rm dspace-cli create-administrator -e test@test.edu -f admin -l user -p admin -c en
```

Comment:
I propose to change this as well as following line to unify mails. 
e.g. admin@test.edu and user@test.edu